Makefile

```diff
@@ -47,21 +47,22 @@ endif
 COMPIL_DATE := $(shell date +'%Y-%m-%d')
 COMMIT_DATE := $(shell git show -s --date=short --pretty=format:'%cd')
 COMMIT_ID := $(shell git rev-parse --short HEAD)
-COMMIT_TITLE := $(shell git log -1 --no-merges --pretty="%s" | sed "s/\"/\\\"/g")
-COMMIT_COMMENT:= $(shell git log -1 --no-merges --pretty="%b" | sed "s/\"/\\\"/g")
+COMMIT_TITLE := $(shell git log -1 --pretty="%s" | sed "s/\"/\\\"/g")
+COMMIT_COMMENT:= $(shell git log -1 --pretty="%b" | sed "s/\"/\\\"/g")
 BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
 VERSION := $(shell awk -F\" '/er =/ {print $$2}' lib/version.txt)
 SOURCE := $(shell git remote get-url origin | sed -n 's|^.*github.com.\([^/]\+/[^/.]\+\)\(.git\)\?|\1|p')
 OCAMLV := $(shell ocaml --version)
 
-lib/version.ml: lib/version.txt
+lib/version.ml:
 	@cp lib/version.txt $@
 	@printf 'let branch = "$(BRANCH)"\n' >> $@
 	@printf 'let src = "$(SOURCE)"\n' >> $@
 	@printf 'let commit_id = "$(COMMIT_ID)"\n' >> $@
 	@printf 'let commit_date = "$(COMMIT_DATE)"\n' >> $@
 	@printf 'let compil_date = "$(COMPIL_DATE)"\n' >> $@
 	@printf 'Generating $@… Done.\n'
+.PHONY: lib/version.ml
```

Comment:
Can you please just merge this new line with the next one `.PHONY: lib/version.ml build fmt gwd install`. Thank you.

Reply:
This  was  resolved by last PR update, ready to merge in master branch.